src/components/Form/Submit.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "react";
+import { useFormikContext } from "formik";
+import { Button, ButtonProps as BsButtonProps, Spinner } from "react-bootstrap";
+
+export interface SubmitProps extends BsButtonProps {
+  withSpinner?: boolean;
+}
+
+export const Submit: FC<SubmitProps> = ({
+  withSpinner,
+  ...props
+}: SubmitProps) => {
+  const { isSubmitting } = useFormikContext();
+  const disabled = isSubmitting || props.disabled;
+
+  return (
+    <Button type="submit" {...props} disabled={disabled}>
+      {withSpinner && isSubmitting ? (
+        <Spinner size="sm" animation="border" className="mr-1" />
+      ) : null}
+      {props.children}
+    </Button>
```

Comment:
I'm open to adding a `SubmitButton`, but I'm not sure about the `Spinner`. That will differ based on given designs, so might be better to leave it out and let the consumer extend it if they need to.